File: SnackBar/app/src/main/java/com/example/snackbar/MainActivity.java
package com.example.snackbar;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.google.android.material.snackbar.Snackbar;

public class MainActivity extends AppCompatActivity {

    Snackbar snackbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        View view = findViewById(R.id.activity_main);

        snackbar = Snackbar.make(view, "Bem vindo", Snackbar.LENGTH_INDEFINITE);
        snackbar.setDuration(5000);
        snackbar.show();
    }
}

File: IdadeDoCachorro/app/src/main/java/com/example/idadedocachorro/MainActivity.kt
package com.example.idadedocachorro

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)



        button_calcula.setOnClickListener{

            val idade_cachorro = edit_Idade.text.toString()

            val intent = Intent(applicationContext, IdadeCachorroActivity::class.java)

            intent.putExtra("idade", idade_cachorro)



            startActivity(intent)

        }



    }


}

